src/main/java/org/testng/TestRunner.java

```diff
@@ -1557,8 +1577,8 @@ public void onConfigurationSuccess(ITestResult itr) {
     }
   }
 
-  public void setMethodInterceptor(IMethodInterceptor methodInterceptor) {
```

Comment:
Yes, I pointed this out in my initial review:

https://github.com/cbeust/testng/pull/625/